django/views/debug.py

```diff
@@ -544,6 +545,17 @@ def get_exception_traceback_frames(self, exc_value, tb):
                 pre_context = []
                 context_line = "<source code not available>"
                 post_context = []
+            if PY311 and not sys._xoptions.get("no_debug_ranges", False):
```

Comment:
There is no need to check `no_debug_ranges`. `start_column` and `end_column` are `None` when `-X no_debug_ranges` is used.